webapp/client/public/css/components.css

```diff
@@ -518,31 +526,29 @@ input[type="radio"] + label::before {
     vertical-align: bottom;
     width: 30px;
     height: 30px;
+    min-width: 30px;
```

Comment:
Yes, I thought so, too. But thank you for asking that again, I know found out why: https://stackoverflow.com/a/51069379
Basically, because the parent element has `display: flex` the width attribute doesn't work as expected anymore and instead is used as `flex-basis` - setting `display: inline-block` as we did for the ::before element also doesn't change that behavior.

Reply:
Thank you for that explanation. As far as I understand it now ([after reading this article linked in the stackoverflow](https://mastery.games/post/the-difference-between-width-and-flex-basis/) comment), we have a flex-basis of 30px and the flex container shrinks and therefore the elements shrink as well. But because we set min-width to 30px they can not shrink below the threshold. 

That said, we could just use flex-basis instead of width then, correct?

Reply:
Yes we could. I just find min-width more intuitive?

Reply:
We talked about it and only using flex-basis could raise problems in case the radio button is not within a flexbox, so we're keeping width.